### PR TITLE
Fix inconsistent default learning rate in tensorflow.keras.optimizers.Adadelta()

### PR DESCRIPTION
**Pull request by ilhamfp**
Fixing [#31024 issue](https://github.com/tensorflow/tensorflow/issues/31024) of inconsistencies between tensorflow.keras.optimizers.Adadelta() and keras.optimizers.Adadelta().

### CODE REVIEW

**Comment by ilhamfp** [comment]
I signed it!

**Comment by ilhamfp** [comment]
I signed it!

**Comment by tanzhenyu** [comment]
Thanks for the PR! We have made a decision to make 0.001 as default learning rate for most of our optimizers, and put release notes regarding the changes.

**Comment by ilhamfp** [comment]
@tanzhenyu does that mean Keras is the one that needs to change their learning rate?

**Comment by tanzhenyu** [comment]
Yeah I had a PR https://github.com/keras-team/keras/pull/12841 but was reverted due to some weird issues, still working on it.

**Comment by ilhamfp** [comment]
@tanzhenyu  Ah well, okay then. Just let me know if I can help you with anything. :)

**Comment by ilhamfp** [comment]
@tanzhenyu Does that mean the learning rate for Adadelta() on [this one](https://github.com/tensorflow/tensorflow/blob/r1.14/tensorflow/python/keras/optimizers.py) need to be changed to 0.001?
